apps/ledger-live-mobile/src/components/FirmwareUpdateBanner.tsx

```diff
@@ -250,7 +250,14 @@ const FirmwareUpdateBanner = () => {
           <Button
             type="main"
             outline={false}
-            onPress={onExperimentalFirmwareUpdate}
+            onPress={
+              newFwUpdateUxFeatureFlag?.enabled
+                ? onExperimentalFirmwareUpdate
+                : () => {
+                  setShowBatteryWarningDrawer(false);
+                  setShowUnsupportedUpdateDrawer(true);
+                }
+            }
```

Comment:
I would personally move this to a callback with dependencies on the enabled flag and other function, but I don't care enough anymore

Reply:
Done